tests/unit/ai/review/test_context_parse.py

```
@@ -0,0 +1,417 @@
+    changed_files = parse_changed_files(name_status=name_status, numstat=numstat)
+
+    assert_that(changed_files).is_length(1)
+    assert_that(changed_files[0].path).is_equal_to(expected_path)
+    assert_that(changed_files[0].status).is_equal_to(ChangedFileStatus(expected_status))
+    assert_that(changed_files[0].previous_path).is_equal_to(expected_previous)
+
+
+def test_split_unified_diff_by_file_supports_quoted_paths() -> None:
+    """Quoted diff headers parse paths containing spaces."""
+    diff_text = (
+        'diff --git "a/foo bar.py" "b/foo bar.py"\n' "+++ b/foo bar.py\n" "+change\n"
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

(ISC001)